source/_components/sensor.sensirion_sht31.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: "Sensirion SHT31 Smart-Gadget"
+description: "Instructions on how to integrate Sensirion SHT31 Smart-Gadget within Home Assistant."
+date: 2018-06-18 17:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: sensirion.png
+ha_category: Sensor
+ha_release: 0.72
+ha_iot_class: "Local Polling"
+---
+
+The `sensirion_sht31 ` sensor platform allows you to get the current battery, temperature and humidity from a Sensirion SHT31 Smart-Gadget device.
+
+## {% linkable_title Configuration %}
+
+To use your Sensirion SHT31 Smart-Gadget sensor in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: sensirion_sht31
+    mac: DEVICE_MAC
+    name: SHT31 Smart-Gadget
+    monitored_conditions:
+      - battery
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.